docs/advanced/tracking.story.mdx

```diff
@@ -0,0 +1,67 @@
+import { Meta, Intro, Story } from '../../.storybook/components';
+
+<Meta title="Advanced/Build-in Tracking" />
+
+# Build-in Tracking
+
+<Intro>
+  Working along with{' '}
+  <a href="https://github.com/sumup-oss/collector">Collector</a>, button-like
+  components and meanful UI operation in Circuit UI will come along with
```

Comment:
```suggestion
  <a href="https://github.com/sumup-oss/collector">Collector</a>, interactive
  components in Circuit UI will come along with
```

Reply:
I'm thinking about components like "tabs" and "pagination". If we want to track the interaction every time a user clicks on it, what `label` should we set as default? For example, we have 3 tabs rendered, should we set different `label` for each of these 3 different tabs? 
We could also not set any default tracking and leave this to the consumer. They will add the tracking by themselves. In this case, we only add default tracking to "button-like" component and not every interactive component. That's why I put the word here.

Reply:
I was under the assumption that we don't provide a default `label`, only a default `component` value. The consumer needs to pass at least a `label` to enable tracking. 

This way, we just need to ensure that unique labels can be set on the Tabs and Pagination subcomponents.

Reply:
Ahhh true! I messed up the concept of component and label. Got it. thank you!